aws/resource_aws_lb_listener_rule_test.go

```diff
@@ -96,6 +96,82 @@ func TestAccAWSLBListenerRule_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSLBListenerRule_forwardWeighted(t *testing.T) {
+	var conf elbv2.Rule
+	lbName := fmt.Sprintf("testrule-weighted-%s", acctest.RandStringFromCharSet(13, acctest.CharSetAlphaNum))
```

Comment:
Ah, this was bad advice, because load balancer names have a maximum length of 32 characters and the string generated by `acctest.RandomWithPrefix("testrule-weighted")` is too long.
`lbName := fmt.Sprintf("testrule-weighted-%s", acctest.RandString(13))` is the best option.